android/build.gradle

```diff
@@ -13,7 +13,7 @@ buildscript {
 }
 
 rootProject.allprojects {
-    def token = System.getenv('SDK_REGISTRY_TOKEN')
+    def token = project.property('MAPBOX_DOWNLOADS_TOKEN')
```

Comment:
This doesn't return environment variables anymore, does it?

Reply:
You can set Gradle project properties using environment variables: https://docs.gradle.org/current/userguide/build_environment.html#sec:project_properties

Reply:
Thanks!!! It's work in -> /snap/flutter/common/flutter/.pub-cache/hosted/pub.dartlang.org/mapbox_gl-0.12.0/android/build.gradle   (Ubuntu)